src/GovUk.Education.ExploreEducationStatistics.Content.Model/Extensions/ReleaseAmendmentExtensions.cs

```diff
@@ -0,0 +1,144 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Model.Extensions
+{
+    public static class ReleaseAmendmentExtensions
+    {
+        private static readonly string FilterPattern = string.Join(
```

Comment:
I thought this was a bit strange and was wondering why we don't just call it `CommentFilterPattern` or have a comment further down like `// Strip comments` in `UpdateAmendmentContent`.

I see now with the `string.Join` how you have made this generic, expecting to filter more elements of the content in future. Quite nice I guess, if that happens!